src/transmitter/index.ts

```diff
@@ -69,6 +70,30 @@ export async function sendWorkloadMetadata(payload: IWorkloadMetadataPayload): P
     }
 }
 
+export async function sendWorkloadAutoImportPolicy(payload: WorkloadAutoImportPolicyPayload): Promise<void> {
+  try {
+    logger.info(
+      { userLocator: payload.userLocator, cluster: payload.cluster, agentId: payload.agentId },
+      'attempting to send workload auto-import policy',
+    );
+
+    const { response, attempt } = await retryRequest('post', `${upstreamUrl}/api/v1/policy`, payload);
```

Comment:
Do we want to wait for #664 to be merged before we merge this one, and we can use the queue? Or do we want to skip the queue for this payload?

Reply:
Another things, does homebase has `/api/v1/policy` route? Or this PR is actually blocked until we add `/api/v1/policy` into homebase?

Reply:
Doesn't exist yet but doesn't have to be blocking since it will print an error that it can't send the policy only once on deployment (assuming we can tolerate that until the Homebase endpoint is added)